mediation-old/src/main/java/com/wso2telco/dep/mediator/impl/payment/AmountRefundHandler.java

```diff
@@ -209,6 +213,28 @@ private String makeRefundResponse(String responseStr, String requestid,
 
 	}
 
+	/**
+	 * This method extracts userId from payload and resource url and passed to
+	 * validate whether they are same
+	 */
+	private void compareMsisdn() {
```

Comment:
Comparing "msisdn" behaviour is comoon for both Refund and Charge operations. Hence can we move this into common location

Reply:
Yes. Will we do so.